### PR TITLE
Fixes #36963 - Table index new button far in large screens

### PR DESCRIPTION
**Pull request by MariaAga**
Before
Screenshot from 2023-12-05 10-53-09
After
Screenshot from 2023-12-05 10-51-45

### CODE REVIEW

File: webpack/assets/javascripts/react_app/components/SearchBar/SearchBar.scss [hunk 1]
```diff
@@ -15,4 +15,7 @@
   .pf-c-dropdown.pf-m-align-right {
     width: unset;
   }
+  .pf-c-dropdown { 
+    width: unset; // just in case Katello overrides it
```
Comment (MariaAga) [inline webpack/assets/javascripts/react_app/components/SearchBar/SearchBar.scss:19]: It shouldnt now https://github.com/Katello/katello/pull/10748

**Review by kmalyjur — APPROVED** [review]
Awesome! I was maybe thinking that 8px looks better than 16px ([according to Patternfly](https://v4-archive.patternfly.org/v4/components/button/design-guidelines) it can be 8 if it's in the toolbar next to the icon button), but it's probably just personal preference. Other than that LGTM.

**Comment by MariaAga** [comment]
Thanks! I'm using the design from https://www.sketch.com/s/7c65523b-3b2b-4fbd-b785-9d323dac6b95/a/52Ryg1P 
Which I think is using 16px

**Review by ofedoren — APPROVED** [review]
Thanks, @MariaAga, can confirm that it fixes the issue for 1440p monitors :)

**Pull request merged**